packages/bootstrap-vue-3/src/utils/inspect.ts

```diff
@@ -19,3 +19,11 @@ export const isNumeric = (value: unknown): boolean => /^[0-9]*\.?[0-9]+$/.test(S
  */
 export const isPlainObject = (obj: unknown): obj is Record<PropertyKey, unknown> =>
   Object.prototype.toString.call(obj) === '[object Object]'
+
+/**
+ * Strict object type check. Only returns true for plain JavaScript objects
+ *
+ * @param obj
+ * @returns
+ */
+export const isNull = (value: unknown): Boolean => value === null
```

Comment:
TypeScript magic will then tell you what the value is (type predicates)
```suggestion
export const isNull = (value: unknown): value is null => value === null
```